app/javascript/react/components/NetflixTile.js

```diff
@@ -1,10 +1,13 @@
 import React from 'react'
+import { Link } from "react-router-dom"
 
 const NetflixTile = (props) => {
 
   return (
     <div>
-      {props.title}
+      <Link to={`netflix_shows/${props.id}`} >
```

Comment:
Does this need a `/` at the start?  Looks like it should be 
```js
<Link to={`/netflix_shows/${props.id}`} >
```

If this is working you can disregard this

Reply:
It was working when we were testing it.